Backend/TexasCodeEm/src/main/java/com/epiclabs/texascodeem/api/Player.java

```diff
@@ -0,0 +1,40 @@
+package com.epiclabs.texascodeem.api;
+
+public class Player {
+
+    private final String name;
+    private final String id;
+    private int stackSize;
+
+    private final int DEFAULT_STACK_SIZE = 5000;
+
+    public Player(String name, String id) {
+        this.name = name;
+        this.id = id;
+        this.stackSize = DEFAULT_STACK_SIZE;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public int getStackSize() {
+        return stackSize;
+    }
+
+    public void setStackSize(int stackSize) {
+        this.stackSize = stackSize;
```

Comment:
I'd like to see consistency on whether on not to use "this" to refer to class variables. Thanks.